02.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -718,6 +719,56 @@ For the requests matching a `$csp` rule, we will strengthen response's security
 * `@@||example.org/page/*$csp` — disables all the `$csp` rules on all the pages matching the rule pattern.
 * `||example.org^$csp=script-src 'self' 'unsafe-eval' http: https:` — disables inline scripts on all the pages matching the rule pattern.
 
+<a id="cookie-modifier"></a>
+##### **`cookie`**
+
+The `$cookie` modifier completely changes rule behavior. Instead of blocking a request, this modifier makes us suppress or modify the `Cookie` and `Set-Cookie` headers.
+
+> **Multiple rules matching a single request**
+> In case if multiple `$cookie` rules match a single request, we will apply each of them one by one.
+
+### `$cookie` syntax
+The rule syntax depends on whether we are going to block all cookies or to remove a single cookie. The rule behavior can be changed with `maxAge` and `sameSite` modifiers.
+
+* `||example.org^$cookie=NAME;maxAge=3600;sameSite=lax` -- every time AdGuard encounters a cookie called `NAME` in a request to `example.org`, it will do the following:
+  
+  * Set its expiration date to current time plus `3600` seconds
+  * Makes the cookie use [Same-Site](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#SameSite_cookies) "lax" strategy.
+* `||example.org^$cookie` -- blocks ALL cookies set by `example.org`. This is an equivalent to setting `maxAge` to zero.
+* `||example.org^$cookie=NAME` -- blocks a single cookie named `NAME`
+* `||example.org^$cookie=/regular_expression/` -- blocks every cookie that matches a given regular expression
+
+> **Important:** in the case of a regular expression matching, two characters must be escaped: comma (`,`) and (`$`). Use (`\`) for it. For example, escaped comma looks like this: `\,`.
+
+`$cookie` rules are not affected by regular exception rules (`@@`) unless it's a `$document` exception. In order to disable a `$cookie` rule, the exception rule should also have a `$cookie` modifier. Here's how it works:
+
+* `@@||example.org^$cookie` -- unblocks all cookies set by `example.org`
+* `@@||example.org^$cookie=NAME` -- unblocks a single cookie named `NAME`
+* `@@||example.org^$cookie=/regular_expression/` -- unblocks every cookie matching a given regular expression
+
+> **Limitations**
+> `$cookie` rules support a limited list of modifiers: `domain`, `~domain`, `important`, `third-party`, `~third-party`.
+
+### Implementation details
```

Comment:
This part is a part of the task for the developers. We should simply remove it from the documentation.

Reply:
removed task for the developers from KB